components/server/src/workspace/workspace-starter.ts

```diff
@@ -171,6 +173,17 @@ export class WorkspaceStarter {
             const resp = (await manager.startWorkspace({ span }, startRequest)).toObject();
             span.log({ "resp": resp });
 
+            this.analytics.track({ 
+                userId: user.id, 
+                event: "workspace-started", 
```

Comment:
a definition of constants for event names would make sense, or not? at least to see the names being aligned.

Reply:
dunno. I could also imagine those names being generated in some cases.